src/Stepper/VerticalStep.jsx

```diff
@@ -0,0 +1,336 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+import Mixins from '../mixins';
+
+import {getMuiTheme} from '../styles';
+
+const Step = React.createClass({
+  propTypes: {
+    children: PropTypes.node,
+
+    /**
+     *Override the inline-styles of div which contains all the children include control button groups
+     */
+    childrenWrapperStyle: PropTypes.object,
+    /**
+     *Override the inline-styles of connector line
+     */
+    connectorLineStyle: PropTypes.object,
+    /**
+     *An array of node for handling moving or canceling steps
+     */
+    controlButtonsGroup: PropTypes.arrayOf(PropTypes.node),
+    /**
+     *Override the inline-styles of div wrapper which contains control buttons group
+     */
+    controlButtonsGroupWrapperStyle: PropTypes.object,
+    /**
+     *If true, the step is active
+     */
+    isActive: PropTypes.bool,
+    /**
+     *If true, the step is completed
+     */
+    isCompleted: PropTypes.bool,
+    /**
+     *If true, the step is the last one
+     */
+    isLastStep: PropTypes.bool,
+    /**
+     *If true, the header of step is hovered
+     */
+    isStepHeaderHovered: PropTypes.bool,
+
+    /**
+     *Callback function that is fired when the header of step is hovered
+     */
+    onHeaderStepHover: PropTypes.func,
+    /**
+     *Callback function that is fired when the header of step is touched
+     */
+    onHeaderStepTouch: PropTypes.func,
+    /**
+     *The string will be shown in the left avatar of step header
+     */
+    orderStepLabel: PropTypes.string,
+    /**
+     *The index of the furthest optional step
+     */
+    previousStepOptionalIndex: PropTypes.number,
+    /**
+     *Override the inline-styles of step container which contains connector line and children
+     */
+    stepContainerStyle: PropTypes.object,
+    /**
+     *Override the inline-styles of step header view (not include left avatar)
+     */
+    stepHeaderStyle: PropTypes.object,
+    /**
+     *Override the inline-styles of step header wrapper (include left avatar)
+     */
+    stepHeaderWrapperStyle: PropTypes.object,
+    /**
+     *The index of step in array of Steps
+     */
+    stepIndex: PropTypes.number,
+    /**
+     *The label of step which shows in step header
+     */
+    stepLabel: PropTypes.string,
+    /**
+     *Customize the step label view
+     */
+    stepLabelView: PropTypes.node,
+  },
+
+  contextTypes: {
+    muiTheme: PropTypes.object,
+    completedIcon: PropTypes.node,
+  },
+
+  childContextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+
+  mixins: [
+    Mixins.StylePropable,
```

Comment:
This mixin was deprecated. This PR will help https://github.com/callemall/material-ui/pull/3349/files.